Steer/DigitizerWorkflow/CMakeLists.txt

```diff
@@ -46,11 +46,10 @@ o2_add_executable(digitizer-workflow
                                         O2::PHOSSimulation
                                         O2::CPVSimulation
                                         O2::TOFSimulation
-					                    O2::TOFCalibration
+                                        O2::TOFCalibration
                                         O2::TOFReconstruction
                                         O2::TOFWorkflowUtils
                                         O2::TPCSimulation
-                                        O2::TPCWorkflow
```

Comment:
@sawenzel we can remove the dependency now, but we can also leave it if it's planned to move the TPC specs to `O2::TPCWorkflow` anyhow. Depending on the plans for cleaning up the digitizer workflow.

Reply:
Yes, I think we should move these things into the proper place. So let's leave it.